src/pyupgrade/plugins/rewrite_unicode_literal.rs

```diff
@@ -0,0 +1,30 @@
+use rustpython_ast::Expr;
+
+use crate::ast::types::Range;
+use crate::autofix::Fix;
+use crate::checkers::ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+pub fn rewrite_unicode_literal(
+    checker: &mut Checker,
+    expr: &Expr,
+    value: &str,
+    kind: &Option<String>,
+) {
+    if let Some(const_kind) = kind {
+        if const_kind == "u" {
+            let mut check = Check::new(CheckKind::RewriteUnicodeLiteral, Range::from_located(expr));
+            if checker.patch(check.kind.code()) {
+                let mut new_str = value.to_string();
+                new_str.insert(0, '"');
+                new_str.push('"');
```

Comment:
Can probably be written a bit more cleanly as:

```rust
let mut contents = String::new(value.len() + 2);
contents.push('"');
contents.push_str(value);
contents.push('"');
```

Actually, though, I think this may need to be adjusted to respect the quotation style.

For example:
```
u"""Abc"""
```

Here, we need to respect the triple quotes.

There's a function to help with this: `src/pydocstyle/helpers.rs#leading_quote`. You'll need to use `checker.locator` to grab the entire expression, not just the `value` contents though.